src/test/java/step2/domain/LottoNumTest.java

```diff
@@ -0,0 +1,56 @@
+package step2.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import step2.domain.Lotto.LottoNum;
+import step2.domain.Lotto.LottoNums;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+public class LottoNumTest {
+
+    @Test
+    @DisplayName("로또 넘버 객체 생성")
+    void lottoNum() {
+        assertThat(new LottoNum(2)).isEqualTo(new LottoNum(2));
+    }
+
+    @Test
+    @DisplayName("로또 넘버 객체 예외처리")
+    void lottoNum_underNumAndOverNumThrows() {
+        assertThatIllegalArgumentException()
+                .isThrownBy(() -> new LottoNum(0))
+                .withMessageMatching("로또 번호는 1 ~ 45까지의 숫자만 가능합니다.");
+
+        assertThatIllegalArgumentException()
+                .isThrownBy(() -> new LottoNum(46))
+                .withMessageMatching("로또 번호는 1 ~ 45까지의 숫자만 가능합니다.");
+    }
+
+    @Test
+    @DisplayName("로또 번호 비교")
+    void lottoNums_lottoNumsCompareToLottoList() {
+        LottoNums lottoNums = new LottoNums(new LottoNums.Builder()
+                .lottoNum(new LottoNum(1))
+                .lottoNum(new LottoNum(2))
+                .lottoNum(new LottoNum(16))
+                .lottoNum(new LottoNum(19))
+                .lottoNum(new LottoNum(30))
+                .lottoNum(new LottoNum(32)));
```

Comment:
```suggestion
        LottoNums lottoNums = new LottoNums(1, 2, 16, 19, 30, 32);
```
위와 같이 생성할 수 있는 생성자를 추가하는 것은 어떨까?